runtime/laos/src/configs/xcm_config.rs

```diff
@@ -294,4 +312,25 @@ mod tests {
 			"0x6d6F646c70792F78636D63680000000000000000"
 		);
 	}
+
+	#[test]
+	fn only_teleport_native_contains_only_native_asset() {
+		let assets: Vec<Asset> = vec![(HereLocation::get(), Fungible(1_000)).into()];
+
+		// The first parameter passed to contains may be any location as it's not used by the
+		// function. We use HereLocation for simplicity.
+		assert!(OnlyTeleportNative::contains(&(HereLocation::get(), assets)));
+	}
+
+	#[test]
+	fn only_teleport_native_contains_not_all_assets_are_native() {
+		let assets: Vec<Asset> = vec![
+			(HereLocation::get(), Fungible(1_000)).into(),
+			(RelayLocation::get(), Fungible(1_000)).into(),
```

Comment:
You have to declare `RelayLocation` in this test, as it's been removed in `main` (it wasn't used). Since you're at it, could you please remove `RelayLocation` from `xcm-simulator/src/laosish/configs/xcm_config.rs` as well, please? It's not used either

Reply:
As RelayLocation's not needed, I changed the test to use an arbitrary location instead, due to RelayLocation doesn't have a special meaning in the test, it was just an already declared location.


Removed from ```xcm-simulator/src/laosish/configs/xcm_config.rs``` as well :)